x/subscription/keeper/cu_tracker.go

```diff
@@ -243,10 +220,9 @@ func (k Keeper) RewardAndResetCuTracker(ctx sdk.Context, cuTrackerTimerKeyBytes
 			utils.LogLavaEvent(ctx, k.Logger(ctx), types.MonthlyCuTrackerProviderRewardEventName, map[string]string{
 				"provider":         provider,
 				"sub":              sub,
-				"plan":             subObj.PlanIndex,
 				"tracked_cu":       strconv.FormatUint(trackedCu, 10),
 				"credit_used":      creditToSub.String(),
-				"credit_remaining": subObj.Credit.String(),
+				"credit_remaining": timerData.Credit.String(),
```

Comment:
is it the remaining credit?

Reply:
done. see https://github.com/lavanet/lava/commit/258ba6c80e921517193f08b73232259ec4a942a5